wagtail/wagtailforms/tests.py

```diff
@@ -79,29 +79,6 @@ def test_post_multiple_values(self):
         self.assertIn("baz", submission[0].form_data)
 
 
-class TestPageModes(TestCase):
```

Comment:
This change has been made on master: https://github.com/torchbox/wagtail/commit/af7c1de978a7647d6c2023dda43133d3523bb103